### ISSUE TITLE
Update openai script

### ISSUE THREAD

**Issue by lydgate** [Gryphire/wisperbot#37]
Looks like openai module has changed, need to run `openai migrate` to update the code.

**Comment by lydgate** [comment]
Fixed in 282fdade8b1f7158587ddfef37202fa5474b76ad

**Comment by Gryphire** [comment]
May have to do with the module changing, but transcription is currently broken in [628e66d](https://github.com/Gryphire/wisperbot/commit/628e66dc28c101b2ee9c15a8d3b678491d782a49). 

Getting the following error:

```
2024-03-15 08:39:44,176 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/Users/nastasia/surfdriveUT/wisperbot/wisperbotv2_15032024.py", line 367, in transcribe
    transcript = client.audio.transcribe('whisper-1',audio_file).pop('text')
AttributeError: 'Audio' object has no attribute 'transcribe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/telegram/ext/_application.py", line 1124, in process_update
    await coroutine
  File "/usr/local/lib/python3.9/site-packages/telegram/ext/_handler.py", line 141, in handle_update
    return await self.callback(update, context)
  File "/Users/nastasia/surfdriveUT/wisperbot/wisperbotv2_15032024.py", line 431, in get_voicenote
    transcript = await chat.transcribe(filepath)
  File "/Users/nastasia/surfdriveUT/wisperbot/wisperbotv2_15032024.py", line 373, in transcribe
    except openai.InvalidRequestError:
AttributeError: module 'openai' has no attribute 'InvalidRequestError'
```

**Comment by lydgate** [comment]
Actually, I can probably avoid this problem and make error handling more general for transcription, could you please try again with the latest from #29? Tried to fix in 78fb0fe3b35a6f7c52d5c30e13fd558043619a7a

**Comment by Gryphire** [comment]
Yep, I upgraded the openai module the day before yesterday (because the ‘from openai import OpenAI’ line threw an error and suggested to update). 
I'm running 1.14.

**Comment by Gryphire** [comment]
Fixed in branch 29's latest push, [cc9c128](https://github.com/Gryphire/wisperbot/commit/cc9c128c49a7c31117079ea8a0f645acdebe4692).

**Comment by Gryphire** [comment]
Fixed in branch 29's latest push, [cc9c128](https://github.com/Gryphire/wisperbot/commit/cc9c128c49a7c31117079ea8a0f645acdebe4692).
